pkg/app/app_reconcile.go

```diff
@@ -190,12 +190,14 @@ func (a *App) resetLastDeployStartedAt() {
 func (a *App) reconcileInspect() error {
 	inspectResult := a.inspect().WithFriendlyYAMLStrings()
 
-	a.app.Status.Inspect = &v1alpha1.AppStatusInspect{
-		Stdout:    inspectResult.Stdout,
-		Stderr:    inspectResult.Stderr,
-		ExitCode:  inspectResult.ExitCode,
-		Error:     inspectResult.ErrorStr(),
-		UpdatedAt: metav1.NewTime(time.Now().UTC()),
+	if inspectResult != (exec.CmdRunResult{}) {
```

Comment:
would be nice to add IsEmpty() bool on CmdRunResult

Reply:
I was just looking for an alternative to do the comparison here. Thanks :)

Reply:
Done.